Cargo.toml

```diff
@@ -102,6 +102,7 @@ members = [
     "examples/counter_advanced",
     "examples/canvas",
     "examples/drop_zone",
+	"examples/markdown",
```

Comment:
Extra indentation?

Reply:
Not intended, my editor wasn't properly configured and inserted one tab, instead of four spaces. This was correctly displayed for me, although it isn't.